configuration/pih/pih-config-haiti.json

```diff
@@ -70,6 +70,18 @@
           "mothersName": {
             "required": true
           }
+        },
+        "contactInfo": {
+            "phoneNumber": {
+              "regex": "(^\\d{8}$)|(^\\d{4}(?:\\)|[-|\\s])?\\s*?\\d{4}$)"
+            }
+        },
+        "contactPerson": {
```

Comment:
I had to add support in PIH Core for adding regex support "contactPerson", that change will be coming in another PR.